collectors/freeipmi.plugin/freeipmi_plugin.c

```diff
@@ -1619,6 +1619,14 @@ int parse_outofband_driver_type (const char *str)
     return (-1);
 }
 
+int host_is_local(const char *host)
+{
+    if (host && (!strcmp(host, "localhost") || !strcmp(host, "127.0.0.1") || !strcmp(host, "::1")))
```

Comment:
We are using only one of the possible representations for IPV6 `loopback`  here, please, add also the [unicast representation](https://tools.ietf.org/html/rfc4291#section-2.5.3) and the [IPV6 notation](https://tools.ietf.org/html/rfc4291#section-2.4).

Reply:
Well, these checks are for the _most common_ cases, not for every corner case. If the user misconfigures the plugin he gets an error message from the `libipmimonitoring` library itself (but it's not as clear as our error message). My intention was to shorten [the list of addresses]( https://git.savannah.gnu.org/cgit/freeipmi.git/tree/common/miscutil/network.c#n271) to check.